src/NServiceBus.Gateway.AcceptanceTests/When_sending_a_message_to_another_site.cs

```diff
@@ -12,9 +12,9 @@
     public class When_sending_a_message_to_another_site : NServiceBusAcceptanceTest
     {
         [Test]
-        public async Task Should_be_able_to_reply_to_the_message()
+        public Task Should_be_able_to_reply_to_the_message()
         {
-            await Scenario.Define<Context>()
+            return Scenario.Define<Context>()
```

Comment:
@SimonCropp This used to not work in previous versions of NUnit. Is this fixed then? // cc @Particular/nservicebus-maintainers